addon/chrome/content/preferences/prefs-ce.js

```diff
@@ -1039,69 +1039,66 @@ class PrefWindow extends MozXULElement {
           <spacer anonid="spacer" flex="1"/>
           <button dlgtype="cancel" class="dialog-button" icon="cancel"/>
           <button dlgtype="accept" class="dialog-button" icon="accept"/>`;
-    return `
+    else
+      return `
           <button dlgtype="extra2" class="dialog-button" hidden="true"/>
           <spacer anonid="spacer" flex="1"/>
           <button dlgtype="accept" class="dialog-button" icon="accept"/>
           <button dlgtype="extra1" class="dialog-button" hidden="true"/>
           <button dlgtype="cancel" class="dialog-button" icon="cancel"/>
           <button dlgtype="help" class="dialog-button" hidden="true" icon="help"/>
-          <button dlgtype="disclosure" class="dialog-button" hidden="true"/>`;
+          <button dlgtype="disclosure" class="dialog-button" hidden="true"/>`;          
   }
 
   connectedCallback() {
     if (this._initialized) {
       return;
     }
 
-    function updateAttribute(name, value) {
```

Comment:
Please Don't undo all my changes :)
The only bug is in updateAttribute function just replace it with this code and prefs window should work

**new function**
```js
    const updateAttribute = (name, value) => {
      if (!this.hasAttribute(name)) {
        this.setAttribute(name, value);
      }
    };
```

**diff**
```diff
-    function updateAttribute(name, value) {
-      if (this.hasAttribute(name)) {
+    const updateAttribute = (name, value) => {
+      if (!this.hasAttribute(name)) {
         this.setAttribute(name, value);
       }
-    }
+    };
```